handlers/auth_handler_test.go

```
@@ -0,0 +1,862 @@
+
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	if err != nil {
+		t.Fatalf("failed to make request: %v", err)
+	}
+
+	return resp
+}
+
+func parseJSONResponse(t *testing.T, resp *http.Response, target interface{}) {
+	defer resp.Body.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)